static/src/js/components/OrderStatus/__tests__/mocks.js

```diff
@@ -74,3 +82,83 @@ export const orderStatusPropsTwo = {
   }
 }
 
+export const orderStatusPropsEsi = {
+  authToken: 'testToken2',
+  order: {
+    id: 7,
+    collections: {
+      esi_orders: [
+        {
+          collection_id: 'TEST_COLLECTION_123',
+          collection_metadata: {
+            dataset_id: 'Test Dataset ID 2'
+          },
+          access_method: {
+            order: {
+              order_status: 'in progress',
+              order_id: [
+                '5000000333461',
+                '5000000333462'
+              ],
+              service_options: {
```

Comment:
This key is no longer necessary, it's an old key from 1.0. I'd rather not go forward with old data -- the best thing here would just be to remove it.

Reply:
This will be handled under the order status work